.azure-pipelines/linux-conda-CI.yml

```diff
@@ -107,6 +118,15 @@ jobs:
       pip install -e .
     displayName: 'install'
 
+  - script: |
+      if [ '$(onnxrt.version)' == 'nightly' ]
+      then
+        pip install -i https://test.pypi.org/simple/ ort-nightly
+      else
+        pip install onnxruntime$(onnxrt.version)
+      fi
+    displayName: 'install onnxruntime again (previous step might remove it)'
+
```

Comment:
Why is this needed twice? If you do this after requirements-dev.txt, then it only needs to be done once.

Reply:
I removed. It did not have any impact. I still get onnxruntime: 0.3.0 on linux and onnxruntime: 0.3.1 on windows.